README.md

```diff
@@ -38,13 +38,31 @@ An [Elastic Email][] adapter for the [Bamboo][] email app for Elixir.
 
 4.  Follow the Bamboo [Getting Started Guide][getting_started].
 
-5.  To use ElasticEmail's [postBack][] functionality you can place a value in
+5.  To use [ElasticEmail's additional api params][email_send] you can place a value in
```

Comment:
`API parameters`, please. It’s a small thing, but will make the text more readable.